src/main/java/com/pizzashop/pizzashop/DatabaseLoader.java

```diff
@@ -0,0 +1,39 @@
+package com.pizzashop.pizzashop;
+
+import com.pizzashop.pizzashop.Models.Topping;
+import com.pizzashop.pizzashop.Repositories.CheesesRepository;
+import com.pizzashop.pizzashop.Repositories.ToppingsRepository;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DatabaseLoader implements CommandLineRunner {
+    private ToppingsRepository toppingsRepository;
+
+    public DatabaseLoader(ToppingsRepository tRepository){
```

Comment:
`toppingRepository` is better than `tRepository`, please avoid abbreviations in naming.